CHANGELOG.md

```diff
@@ -61,6 +61,7 @@
 - In the Helm operator, skip owner reference injection for cluster-scoped resources in release manifests. The Helm operator only supports namespace-scoped CRs, and namespaced resources cannot own cluster-scoped resources. ([#1817](https://github.com/operator-framework/operator-sdk/pull/1817))
 - Package manifests generated with [`gen-csv`](https://github.com/operator-framework/operator-sdk/blob/master/doc/sdk-cli-reference.md#gen-csv) respect the `--operator-name` flag, channel names are checked for duplicates before (re-)generation. ([#1693](https://github.com/operator-framework/operator-sdk/pull/1693))
 - Generated inventory for Ansible-based Operators now sets the localhost's `ansible_python_interpreter` to `{{ ansible_playbook_python }}`, to properly match the [implicit localhost](https://docs.ansible.com/ansible/latest/inventory/implicit_localhost.html). ([#1828](https://github.com/operator-framework/operator-sdk/pull/1828))
+- Fix `Manager.initialize` err scope issue, when err happened in initializing process, return the err instead of nil. ([#1976](https://github.com/operator-framework/operator-sdk/pull/1976))
```

Comment:
```suggestion
- OLM Internal manager initialize is not returning errors. ([#1976](https://github.com/operator-framework/operator-sdk/pull/1976))
```
@estroz @jmrodri have you better ideas? Are you ok with?

Reply:
Brief and to the point, commited!